src/js/Views/RaceTrackView/components/createArrowElement.js

```diff
@@ -0,0 +1,9 @@
+function createArrowElement() {
+  const arrowElement = document.createElement('div');
+  arrowElement.classList.add('forward-icon', 'mt-2');
+  arrowElement.textContent = '⬇️️';
+
+  return arrowElement;
+}
+
+export { createArrowElement };
```

Comment:
```suggestion
export function createArrowElement() {
  return createElement(`<div class="forward-icon mt-2">⬇️️</div>`);
}
```

앞선 내용을 토대로 작성해본다면, 여기도 이렇게 표현할 수 있겠네요!